README.md

```diff
@@ -87,13 +87,13 @@ with:
     package
 ```
 
-### ghcr.io and npm.pkg.github.com packages
+### ghcr.io, nuget.pkg.github.com and npm.pkg.github.com packages
```

Comment:
Since this will be the case for all packages, this section is not really needed anymore, right?

Reply:
Agreed - I've updated the readme to remove this section, and added a rate-limit section in its place